src/cr/cube/matrix.py

```diff
@@ -447,9 +447,11 @@ def blocks(cls, cube_result_matrix, measure_propname=None):
     @lazyproperty
     def _base_values(self):
         """2D ndarray of "body" values from cube-result matrix."""
-        raise NotImplementedError(
-            "`%s` must implement `._base_values`" % type(self).__name__
-        )
+        if self._measure_propname is None:
+            raise NotImplementedError(
+                "`%s` must implement `._base_values`" % type(self).__name__
+            )
+        return getattr(self._cube_result_matrix, self._measure_propname)
```

Comment:
I'm still reading this commit by commit, so this might not be relevant, but:
Could we implement something more explicit here (rather than `getattr`)? I'm thinking something like the `__get__` method on the target class, so we end up with:
```
return self._cube_result_matrix[self._measure_propname]
```
and let the `__get__` mechanism deal with the explicitness, the dictionary of existing measures, handling the proper exceptions, etc.

Reply:
yeah, this went away in the last commit or so. I was pretty sure it would, so I didn't sweat it while I was getting the thing roughed out. Basically, the caller gets the "base_measure" and sends it in, rather than delegating that to the subtotaler. That turns out to be necessary in measure consolidation, so this unpleasantness is gone now :)